mssql-plugin/src/e2e-test/java/io.cdap.plugin/MssqlClient.java

```diff
@@ -124,13 +199,19 @@ private static boolean compareResultSetData(ResultSet rsSource, ResultSet rsTarg
                 String columnTypeName = mdSource.getColumnTypeName(currentColumnCount);
                 int columnType = mdSource.getColumnType(currentColumnCount);
                 String columnName = mdSource.getColumnName(currentColumnCount);
-                switch (columnType) {
+                if (columnType == Types.TIMESTAMP) {
+                    GregorianCalendar gc = new GregorianCalendar(TimeZone.getTimeZone("UTC"));
```

Comment:
Use assertEquals instead. In that case the assertion failure contains the expected and actual values in the assertion failures.